docs/DeveloperGuide.adoc

```diff
@@ -25,10 +25,10 @@
 . Rename the project if you want. Click `Next`
 . Ensure that your `\src` and `\test\java` folders are checked. Keep clicking `Next`
 . Click `Finish`
-. Add JUnit 4 to classpath
+. Add JUnit 5 to classpath
 .. Open any test file in `\test\java` and place your cursor over any `@Test` highlighted in red
 .. Press kbd:[ALT + ENTER] and select `Add 'JUnit4' to classpath`
```

Comment:
There's still a reference to JUnit4 here. Intended?

Reply:
On my side, it says "Add 'JUnit5.3' to classpath"

Reply:
On IDEA 2018.2, it shows as JUnit 5.2 so I'm pretty sure it depends on the version of IDEA you have. 
Should we just change it to be more generic?

Reply:
Yes we can probably follow the "(if `5.2` is not available, choose any `5.x` that is later than `5.2`)" wording used below.

Reply:
Wait actually that's the section header to the part where we instruct them how to get the correct version of JUnit so that should actually be `Add JUnit5.4 to classpath` right?

Reply:
No, I don't think it's the section header.